composer.json

```diff
@@ -37,27 +37,27 @@
     "require": {
         "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
         "dflydev/fig-cookies": "^2.0.1 || ^3.0",
-        "laminas/laminas-cli": "^0.1.5 || ^1.0",
-        "laminas/laminas-diactoros": "^2.11.2",
-        "laminas/laminas-httphandlerrunner": "^2.1",
-        "mezzio/mezzio": "^3.0.2",
-        "psr/container": "^1.0 || ^2.0",
+        "laminas/laminas-cli": "^0.1.5 || ^1.8",
+        "laminas/laminas-diactoros": "^3.0",
```

Comment:
For major dependency updates our policy generally is to expand.
Should be both v2 and v3.